catamount/ops/subgraph_op.py

```diff
@@ -236,6 +236,13 @@ def findAllSourcesSinks(self):
                 if is_sink:
                     self._sinks[op.name] = op
 
+    def getFreeSymbols(self):
+        to_return = super(SubgraphOp, self).getFreeSymbols()
+        loop_iter_name = '{}::iters'.format(self.name)
```

Comment:
Newsha: Question: This seems to imply that you're using subgraphs like loops... Should this be somewhere else?

Reply:
Yes, you're correct. Currently, I'm not using subgraphs exactly correctly. We need to do some refactoring and instantiate IfConditionSubgraphs vs. LoopSubgraphs, and handle their respective symbols appropriately (for a later PR).